exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,13 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (num >= 0) {
+    return "positive";
+  } else {
```

Comment:
Well done Laura! Great solution - just as an alternative, you can get this to work without the else statement:

function negativeOrPositive(number) {
  if (number < 0 ) {
    return "negative";
  }
  return "positive";
}